File: Server/Models/Service.js
const mongoose = require('mongoose')
const Schema = mongoose.Schema

const ServiceSchema = new Schema ({
    objectID:{
        type: String,
        trim: true,
        required: [true, 'Object ID is required']
    },
    name:{
        type: String,
        required: [true,' Name is required'],
        trim: true,
    },
    creatorID:{
        type: String,
        required: [true,'Creator ID is required'],
        trim: true,
    },
    description:{
        type: String,
        trim: true,
        minlength: 1,
        required: [true, 'Description is required']
    },
    serviceDate: Date,
    serviceTime: Time,
    resolverID:{
        type: String,
        trim: true,
    }
})

const Service = mongoose.model('Service', ServiceSchema)

module.exports = Service

File: hello_world/lib/Models/service.dart
class ServiceModel{
  String name;
  String objectID;
  String creatorID;
  String description;
  DateTime date;
  String resolverid;

  ServiceModel({required this.name, required this.objectID ,required this.creatorID,required this.description,required this.date, required this.resolverid});

  factory ServiceModel.fromJson(Map<String, dynamic> json){
    return ServiceModel(
      name: json['Name'],
      objectID: json['ObjectID'],
      creatorID: json['CreatorID'],
      description: json['Description'],
      date: json['Date'],
      resolverid: json['Resolverid'],
    );
  }

  Map<String, dynamic> toJson() => {
    'Name': name,
    'ObjectID': objectID,
    'CreatorID': creatorID,
    'Description': description,
    'Date': date,
    'Resolverid': resolverid,
  };
}